en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md

```diff
@@ -2,7 +2,7 @@
 
 #### Enabling self signup
 
-In a multi-tenanted API Manager setup, self signup to the Developer Portal is enabled by default to all tenants. This will work out-of-the-box for super tenant(`carbon.super`). **But in order to make it work for tenants the following steps should be carried out**.
+In a multi-tenanted API Manager setup, self signup to the Developer Portal will work out-of-the-box for super tenant(`carbon.super`). **But in order to make it work for tenants the following steps should be carried out**.
```

Comment:
let's not write in the future tense. you could change it to "In a multi-tenanted API Manager setup, self signup to the Developer Portal works out-of-the-box for super tenant"